### PR TITLE
HW-55207 Create Transfer Amount Field - require to limit to 14 digits

### PR DESCRIPTION
**Pull request by ymoskovko-epam**
Jira [HW-55207](https://jira.site1.hyperwallet.local/browse/HW-55207)

### CODE REVIEW

File: Transfer/Sources/TransferAmountCell.swift [hunk 1]
```diff
@@ -85,23 +85,33 @@ extension TransferAmountCell: UITextFieldDelegate {
     func textField(_ textField: UITextField,
                    shouldChangeCharactersIn range: NSRange,
                    replacementString string: String) -> Bool {
+        let allowedDigitsCount = 14
         var currentText = textField.text ?? ""
         if string.count > 1 {
             // Paste
             let stringPastedAmount = string.format(with: detailTextLabel?.text)
+            let pastedDigitsOnly = stringPastedAmount.replacingOccurrences(of: "[^0-9]",
```
Comment (jsaini-hw) [inline Transfer/Sources/TransferAmountCell.swift]: why do this? If there's any invalid character other than digits then it shouldn't be allowed to paste

Comment (jsaini-hw) [reply Transfer/Sources/TransferAmountCell.swift]: That's what we want 123,467,543,123.99 to display on UI

**Comment by ymoskovko-epam** [comment]
drawing

**Comment by ymoskovko-epam** [comment]
covered by https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/138

**Pull request closed without merging**